config.ini.example

```diff
@@ -85,7 +85,9 @@ url = https://fahrplan.events.ccc.de/congress/YEAR/Fahrplan/schedule.json
 
 [plugins.dates]
 url = http://www.google.com/calendar/ical/2eskb61g20prl65k2qd01uktis%40group.calendar.google.com/public/basic.ics
+cache = /tmp/dates.cache
 # timedelta = 21
+# announce_minutes = 60,1440
```

Comment:
I realized just now my example was missing the square brackets after the variable name m( Should've been:

```ini
announce_minutes[] = 60
announce_minutes[] = 1440
# announce_minutes[] = [time]
```